CMakeLists.txt

```diff
@@ -1,17 +1,9 @@
-cmake_minimum_required( VERSION 2.8 )
+cmake_minimum_required( VERSION 3.5.1 )
```

Comment:
Is this strictly necessary? If so, fine, if not, it might be annoying to users to work around.